app/services/imports/lettings_logs_import_service.rb

```diff
@@ -174,7 +174,7 @@ def create_log(xml_doc)
                                      0
                                    end
 
-      attributes["offered"] = safe_string_as_integer(xml_doc, "Q20")
+      attributes["offered"] = unsafe_string_as_integer(xml_doc, "Q20")
```

Comment:
safe_string_as_decimal?

Reply:
Not sure if this was a good suggestion, cause we probably wanted the integer to get saved, but it looks like the tests pass